src/App.css

```diff
@@ -123,43 +282,110 @@ ul {
 .proBtn {
   width: 11.5rem;
   height: 2.063rem;
-  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  margin: 0.75rem 0 0.563rem 3.688rem;
   border-radius: 3px;
   background-color: #0290ff;
+  font-family: RobotoSlab, Arial, sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: white;
+  margin-left: 3.8rem;
+  margin-top: 2rem;
+  border: none;
+}
+
+.MainInp {
+  margin-left: 8.8rem;
 }
 
 .inputContainer {
   display: flex;
   flex-direction: row;
-  column-gap: 18%;
-  margin-left: 3%;
 }
 
 .inpBtn {
   width: 11.5rem;
   height: 2.813rem;
+  margin: 1.188rem 0 0 2.125rem;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
   border-radius: 3px;
   background-color: #0290ff;
   color: white;
   font-weight: 400;
   font-size: medium;
-  border: none; /* Remove button border */
+  border: none;
+}
```

Comment:
- [OPTIONAL] It seems the size of your button design doesn't align with the provided design. Please kindly ensure to fix that.👍